docs/users_guide/overview.rst

```diff
@@ -21,17 +21,17 @@ compile MPAS-Ocean:
 
 .. code-block:: bash
 
-    cd MPAS-Model/ocean/develop/
+    cd E3SM-Project/components/mpas-ocean
     # load modules (see machine-specific instructions below)
-    make gfortran CORE=ocean
+    make gfortran
 
 For MALI:
 
 .. code-block:: bash
 
-    cd MPAS-Model/landice/develop/
+    cd E3SM-Project/components/mpas-albany-landice
```

Comment:
This one should be `MALI-Dev/components/mpas-albany-landice`

Reply:
```suggestion
    cd MALI-Dev/components/mpas-albany-landice
```